src/api/handlers/channel.ts

```diff
@@ -471,3 +471,23 @@ export async function isChannelSynced(channelID: string) {
       overwrite.deny !== permission.deny);
   });
 }
+
+/** Adds a recipient to a Group DM using the app granted access token from the `gdm.join` scope */
+export function addGroupDmRecipient(
+  channelID: string,
+  userID: string,
+  token: string,
```

Comment:
```suggestion
  accessToken: string,
```
Much precise and informative. :smile: